docs/concepts/hosts.mdx

```diff
@@ -24,7 +24,7 @@ Observability is always important, but becomes essential in a distributed enviro
 
 #### Logs
 
-Hosts emit unstructured (text) or structured (JSON) logs. The log level and format can be configured for each host [at start time](/docs/reference/host-config).
+Hosts emit unstructured (text) or structured (JSON) logs. The log level and format can be configured for each host [at start time](/docs/cli/wasmcloud).
```

Comment:
Hmnn I'm a bit conflicted on this -- Host Configuration seems like a good place to handle *both* ways of starting the host -- `wash up` and `wasmcloud` binary directly... (and includes ENV vars). 

Are these docs primarily for the starting via `wasmcloud` directly? Maybe we should note that hosts can be started multiple ways?